react/components/ModalDialog/index.js

```diff
@@ -44,7 +45,7 @@ class ModalDialog extends Component {
 
           <Button
             size="small"
-            variation="primary"
+            variation={confirmation.isDangerous ? 'danger' : 'primary'}
```

Comment:
Wouldn't these variations be better in a constant? 🤔 Like:
```
VARIATIONS = {
  PRIMARY: 'primary',
  DANGER: 'danger',
  // other ...
```
```suggestion
            variation={confirmation.isDangerous ? VARIATIONS.DANGER : VARIATIONS.PRIMARY}
```